libs/angular-routing/package.json

```diff
@@ -2,6 +2,10 @@
   "name": "angular-routing",
   "version": "0.1.1",
   "description": "A declarative router for Angular applications",
+  "schematics": "./schematics/collection.json",
+  "ng-add": {
```

Comment:
Nice! I didn't know you could do this.

Reply:
Yeah Angular CLI ng add command has this built-in now.